bin/main.rb

```diff
@@ -1,2 +1,61 @@
 #!/usr/bin/env ruby
-puts 'hello, world'
+class Player
+  attr_reader :name, :letter, :position
+
+  # rubocop:disable Style/ClassVars
+  @@player_letters = []
```

Comment:
- [ ] Its best to not use class variables as recommended by rubocop. Please remove that cop and avoid using class variables.